### ISSUE TITLE
Invites don't show on the app

### ISSUE THREAD

**Issue by smyja** [grishka/Houseclub#82]
Someone sent an invite to my number, the page still remains in the waitlist, isn't there supposed to be an accept invite or is it automatic? Either ways it's been stuck.

**Comment by combrs** [comment]
In my case it was automatic - I simply open app and continue registering (although before it app show "you in the wait list")

**Comment by sshkrlv** [comment]
> In my case it was automatic - I simply open app and continue registering (although before it app show "you in the wait list")

А регистрация прошла успешно, профиль твой открывается?

**Comment by darkzenon** [comment]
Unofficial api always show 2 invite. There is no key for get this information.

**Comment by smyja** [comment]
> In my case it was automatic - I simply open app and continue registering (although before it app show "you in the wait list")

Mine is stuck on the waitlist page

**Comment by smyja** [comment]
> Unofficial api always show 2 invite. There is no key for get this information. 

Okay

**Comment by combrs** [comment]
> > In my case it was automatic - I simply open app and continue registering (although before it app show "you in the wait list")
> 
> А регистрация прошла успешно, профиль твой открывается?

да, всё "работает", но список каналов весьма странный (на арабском, китайском и других подобных языках большая часть). Поиска каналов, как я понимаю - нет.

**Comment by darkzenon** [comment]
> > > In my case it was automatic - I simply open app and continue registering (although before it app show "you in the wait list")
> > 
> > 
> > А регистрация прошла успешно, профиль твой открывается?
> 
> да, всё "работает", но список каналов весьма странный (на арабском, китайском и других подобных языках большая часть). Поиска каналов, как я понимаю - нет.

в этом приложении нет. в api есть поиск. можно юзать clubsearch.io

**Comment by combrs** [comment]
> в этом приложении нет. в api есть поиск. можно юзать clubsearch.io

Это тоже какой-то сторонний ресурс, видимо, там написано "Since approximately February 13, ClubHouse seems to be severely restricting access to event data. "
